### PR TITLE
Radio Buttons for Portarit and Landscape added

### PR DESCRIPTION
**Pull request by salilkhandelwal**
New Feature added with respect to https://musescore.org/en/node/100221

### CODE REVIEW

**Comment by lasconic** [comment]
Please amend your commit message and use Fix #XXXX as instructed in https://musescore.org/en/developers-handbook/git-workflow#Write-some-code!
 http://stackoverflow.com/questions/179123/edit-an-incorrect-commit-message-in-git

File: mscore/pagesettings.ui [hunk 1]
```diff
@@ -501,7 +508,6 @@
   <tabstop>pageGroup</tabstop>
   <tabstop>pageWidth</tabstop>
   <tabstop>pageHeight</tabstop>
```
Comment (lasconic) [inline mscore/pagesettings.ui]: Please include tabstops for your newly added UI elements.

File: mscore/pagesettings.h [hunk 2]
```diff
@@ -67,8 +66,10 @@ class PageSettings : public QDialog, private Ui::PageSettingsBase {
       void pageHeightChanged(double);
       void pageWidthChanged(double);
       void pageOffsetChanged(int val);
+      void on_PortraitButton_clicked();
+      void on_LandscapeButton_clicked();
 
-   public:
+public:
```
Comment (lasconic) [inline mscore/pagesettings.h]: public should keep its alignment. 3 spaces.

File: mscore/pagesettings.ui [hunk 3]
```diff
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>486</width>
+    <width>496</width>
```
Comment (lasconic) [inline mscore/pagesettings.ui]: Is the width change needed?

**Comment by salilkhandelwal** [comment]
While editing using on line editor , will it be alright to commit directly to 100221-pagesetradio ?

**Pull request closed without merging**